### PR TITLE
:sparkles: add longpolling for external tasks

### PR DESCRIPTION
**Pull request by roblen45**
**Changes:**

1. ✨ publish event when ExternalTask created

PR: #183 

---

See the [examples](#example) for inspiration.

## How can others test the changes?

> Describe how others can test your changes (you can remove this section for typo fixes etc.)

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

## Example

<details>
<summary>
:warning: Before merging please click here to expand and follow the guide.
</summary>
<br>

Please use `:twisted_rightwards_arrows:` at the beginning of your merge commit title.

Example 1:

<pre>
<code>
:twisted_rightwards_arrows: :bug: Fix Wrong Text Decoration at ...
</code>
</pre>

To get your commit message, just copy the first part of this pull request.

Example 2:
<pre>
<code>
**Changes:**

- Fixes Wrong Text Decoration at ...
- Fixes some typos
- ...

**Issues:**

Closes #NumberOfFixedIssue

PR: #NumberOfThisPR
</code>
</pre>
</details>

## Emojis

<details>
<summary>
Expand for a list of most used Emojis.
</summary>
<br>

Please prefix your commit messages with an Emoji.

Ref: https://gitmoji.carloscuesta.me/

| Description              | Glyphe               | Emoji  |
|--------------------------|----------------------|--------|
| Bugfix                   | `:bug:`              | 🐛     |
| Configuration releated   | `:wrench:`           | 🔧     |
| Cosmetic                 | `:lipstick:`         | 💄     |
| Dependencies Downgrade   | `:arrow_down:`       | ⬇️     |
| Dependencies Upgrade     | `:arrow_up:`         | ⬆️     |
| Formatting               | `:art:`              | 🎨     |
| Improving Performance    | `:zap:`              | ⚡️     |
| Initial commit           | `:tada:`             | 🎉     |
| Linter                   | `:rotating_light:`   | 🚨     |
| Miscellaneous            | `:package:`          | 📦     |
| New Feature              | `:sparkles:`         | ✨     |
| Refactoring Code         | `:recycle:`          | ♻️     |
| Releasing / Version tags | `:bookmark:`         | 🔖     |
| Removing Stuff           | `:fire:`             | 🔥     |
| Tests                    | `:white_check_mark:` | ✅     |
| Work In Progress (WIP)   | `:construction:`     | 🚧     |

</details>

### CODE REVIEW

File: src/runtime/engine/flow_node_handler/service_task_handler.ts [hunk 1]
```diff
@@ -186,6 +186,9 @@ export class ServiceTaskHandler extends FlowNodeHandler<Model.Activities.Service
 
       await this.persistOnSuspend(serviceTask, token);
 
+      const externalTaskCreatedEventName = `/externaltask/topic/${serviceTask.topic}/created`;
```
[the remark this answers is not in the document]
Comment (roblen45) [reply src/runtime/engine/flow_node_handler/service_task_handler.ts]: Not now. We plan to create an repository especially for that.

[the remark this answers is not in the document]
Comment (ElRaptorus) [reply src/runtime/engine/flow_node_handler/service_task_handler.ts]: @roblen45 @robpal94 Actually, we store these messages in `@process-engine/process_engine_contracts`.
See: 
https://github.com/process-engine/process_engine_contracts/tree/develop/src/runtime/messages/system_events

[the remark this answers is not in the document]
Comment (ElRaptorus) [reply src/runtime/engine/flow_node_handler/service_task_handler.ts]: @roblen45 @robpal94 Actually, we store these messages in `@process-engine/process_engine_contracts`.
See: 
https://github.com/process-engine/process_engine_contracts/tree/develop/src/runtime/messages/system_events

[the remark this answers is not in the document]
Comment (ElRaptorus) [reply src/runtime/engine/flow_node_handler/service_task_handler.ts]: @roblen45 @robpal94 Actually, we store these messages in `@process-engine/process_engine_contracts`.
See: 
https://github.com/process-engine/process_engine_contracts/tree/develop/src/runtime/messages/system_events

[the remark this answers is not in the document]
Comment (ElRaptorus) [reply src/runtime/engine/flow_node_handler/service_task_handler.ts]: @roblen45 @robpal94 Actually, we store these messages in `@process-engine/process_engine_contracts`.
See: 
https://github.com/process-engine/process_engine_contracts/tree/develop/src/runtime/messages/system_events

[the remark this answers is not in the document]
Comment (ElRaptorus) [reply src/runtime/engine/flow_node_handler/service_task_handler.ts]: @roblen45 @robpal94 Actually, we store these messages in `@process-engine/process_engine_contracts`.
See: 
https://github.com/process-engine/process_engine_contracts/tree/develop/src/runtime/messages/system_events

[the remark this answers is not in the document]
Comment (ElRaptorus) [reply src/runtime/engine/flow_node_handler/service_task_handler.ts]: @robpal94 We currently store such routes in `@process-engine/process_engine_contracts`.
See here:
https://github.com/process-engine/process_engine_contracts/blob/develop/src/runtime/messages/event_aggregator_settings.ts

But as @roblen45 pointed out, these will probably be moved to a central repository at some point.

**Pull request merged**